Scripts/script-StopTimeToAssignOnOwnerChange.yml

```diff
@@ -0,0 +1,21 @@
+fromversion: 4.1.0
+commonfields:
+  id: stoptimetoassignonownerchange
+  version: -1
+name: StopTimeToAssignOnOwnerChange
+script: |
+
+  if demisto.args()['old'] == "" and demisto.args()['new'] != "":  # If owner was no-one and is now someone:
+      demisto.executeCommand("stopTimer", {"timerField":"timetoassignment"})
+      demisto.results("Assignment of the incident was successful and so the Time To Assignment timer has been stopped.")
+type: python
+tags:
+- field-change-triggered
+- example
+comment: Stops the "Time To Assign" timer if the owner of the incident was changed.
```

Comment:
args section?

Reply:
As Anar explained under another comment, the args are fetched automatically as the field changes.